src/main/java/org/folio/service/BatchVoucherService.java

```diff
@@ -49,12 +47,15 @@ public void deleteBatchVoucherById(String id, Context vertxContext, Handler<Asyn
 
   public Future<Tx<Map<String, String>>> deleteBatchVoucherById(Tx<Map<String, String>> tx) {
     Promise<Tx<Map<String, String>>> promise = Promise.promise();
-
-    pgClient.delete(tx.getConnection(), BATCH_VOUCHERS_TABLE, tx.getEntity().get(BATCH_VOUCHER_ID), rs -> {
-      logger.info("deletion of batch voucher completed");
+    String batchVoucherId = tx.getEntity().get(BATCH_VOUCHER_ID);
+    log.debug("deleteBatchVoucherById:: Trying to delete batch voucher with id: {}", batchVoucherId);
+    pgClient.delete(tx.getConnection(), BATCH_VOUCHERS_TABLE, batchVoucherId, rs -> {
+      log.info("deleteBatchVoucherById:: deletion of batch voucher completed");
       if (rs.result().rowCount() == 0) {
+        log.warn("deleteBatchVoucherById:: Batch voucher with id '{}' not found", batchVoucherId);
         promise.fail(new HttpException(NOT_FOUND.getStatusCode(), NOT_FOUND.getReasonPhrase()));
       } else {
+        log.info("deleteBatchVoucherById:: Batch voucher with id '{}' deleted", tx.getEntity().get(BATCH_VOUCHER_ID));
```

Comment:
We can use already defined batchVoucherId here

Reply:
I have changed